scylla/src/transport/retry_policy.rs

```diff
@@ -388,22 +388,22 @@ mod tests {
         // Not idempotent
         let mut policy = DefaultRetryPolicy::new().new_session();
         assert_eq!(
-            policy.decide_should_retry(make_query_info(&enough_responses_no_data, false)),
+            policy.decide_should_retry(make_query_info(&enough_responses_with_data, false)),
```

Comment:
This looks like leftovers from #517 and doesn't compile, please remove from this PR.